openzwavemqtt/models/instance_status.py

```diff
@@ -0,0 +1,89 @@
+"""Python wrapper for OpenZWave's MQTT daemon - Model for OZW Instance Status."""
```

Comment:
I think that we can drop the intro on each docstring.
```suggestion
"""Model for OZW Instance Status."""
```

Reply:
fixed